astropy/cosmology/parameters.py

```diff
@@ -150,4 +144,5 @@
                "Table 1 (WMAP + BAO + SN ML).")
 )
 
-available = tuple(k for k in locals() if not k.startswith('_'))
+# If new parameters are added, this list must be updated
```

Comment:
At the risk of being called stubborn, I updated the auto-generation with a simple class-based scheme. It seems pretty clean and introduced a minimal and largely invisible change, but I can do it the other way if that seems more reasonable.